### PR TITLE
feat(Dialog): add new variant confirm

### PR DESCRIPTION
**Pull request by dinarosv**
Added variant `confirm` to Dialog.
See the [Figma file](https://www.figma.com/file/cdtwQD8IJ7pTeE45U148r1/Eufemia---Web?node-id=21007%3A9783) for the different variant design.

### CODE REVIEW

**Review by tujoworker — APPROVED** [review]
Looks good to me now. Just a last question, should we provide the Dialog with `description` or not?

**Comment by dinarosv** [comment]
> Looks good to me now. Just a last question, should we provide the Dialog with `description` or not?

Think we should do that. There were also a few other things I will add, that we talked about, before I merge :)

File: packages/dnb-design-system-portal/src/docs/uilib/components/dialog/prop-table.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 | Properties         | Description                                                                                                                                                                                                                                  |
 | ------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `variant`          | _(optional)_ The dialog variant. Can either be `inform` or `confirm`. Defaults to `inform`.                                                                                                                                                  |
```
Comment (tujoworker) [inline packages/dnb-design-system-portal/src/docs/uilib/components/dialog/prop-table.md:6]: What do you think about `variant="information"` and variant="confirmation" – just to be in line with other "noun" variants.

information = noun
inform = verb

**Review by tujoworker — APPROVED** [review]

File: packages/dnb-design-system-portal/src/docs/uilib/components/dialog/prop-table.md (hunk 1 above)
Comment (tujoworker) [reply packages/dnb-design-system-portal/src/docs/uilib/components/dialog/prop-table.md:6]: It’s good to be in line with UX here. Devs can see the name of UX components. So in this case, the whole word is descriptive witch is actually good.

Comment (dinarosv) [reply packages/dnb-design-system-portal/src/docs/uilib/components/dialog/prop-table.md:6]: I'm convinced 😃 Will change this too!

File: packages/dnb-eufemia/src/components/dialog/parts/DialogAction.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+  /**
+   * For dialog actions, give a custom text for the confirm button.
+   */
+  confirmText?: string
+
+  /**
+   * For variant confirmation, handle the confirm action click.
+   */
+
```
Comment (tujoworker) [inline packages/dnb-eufemia/src/components/dialog/parts/DialogAction.tsx:25]:
```suggestion
```

**Review by tujoworker — APPROVED** [review]

**Pull request merged**